### PR TITLE
changed naming of jobs

### PR DESCRIPTION
**Pull request by mbiarnes**
@psiroky pls review this and let me know if you're fine with the job renaming

### CODE REVIEW

**Comment by psiroky** [comment]
Do we have a naming scheme that all the jobs follow and that we should follow from now on?

File: job-dsls/jobs/kie_release_jobs.groovy [hunk 1]
```diff
@@ -1109,21 +1109,21 @@ job("dash_${DASH_VERSION}_updateVersion") {
...
-                name("kie_${KIE_VERSION}_wbSmokeTestsMatrix")
-                name("kie_${KIE_VERSION}_pushTags")
-                name("kie_${KIE_VERSION}_removeReleaseBranches")
-                name("kie_${KIE_VERSION}_updateToNextDevelopmentVersion")
-                name("kie_${KIE_VERSION}_copyBinariesToFilemgmt")
+                name("createAndPushReleaseBranches-kieReleases-${KIE_VERSION}")
+                name("buildAndDeployLocally-kieReleases-${KIE_VERSION}")
+                name("copyBinariesToNexus-kieReleases-${KIE_VERSION}")
+                name("jbpmTestCoverageMatrix-kieReleases-${KIE_VERSION}")
+                name("serverMatrix-kieReleases-${KIE_VERSION}")
```
Comment (psiroky) [inline job-dsls/jobs/kie_release_jobs.groovy]: Shouldn't this be `kieServerMatrix-...`?

**Comment by psiroky** [comment]
This is the naming scheme suggested by @mbiarnes (next time please do share this kind of info directly in the comment, so that others (who are not part of the e-mail discussions) can have the same info)
```
daily builds:         <jop purpose>-kieAllBuild-<branch>
kie releases:         <job purpose>-kieReleases-<branch>
PR jobs:              <rep>-pullrequest-<branch>
normal build jobs:    <rep>-<branch> 
```
This should be fine as long as we stick to it consistently.

**Comment by mbiarnes** [comment]
@psiroky did much changes and at the end I did git commit --amend - thus a new OR was created 
we can close this one

**Comment by psiroky** [comment]
@mbiarnes please don't create new PRs when changing an already existing PR. We loose all the comments and all the reviews. Let's leave it this time since it's already done, but please think about this for the future PRs.

**Comment by mbiarnes** [comment]
@psiroky yep - it was not my intention to create a new PR - now I know that changing wit --amend the commit message a new PR is created. I will not do it again.

**Comment by psiroky** [comment]
Amending the commit will not create new PR. It will just update the current commit. After that you need to force push into your fork and the PR gets automatically updated.

**Comment by mbiarnes** [comment]
this PR will be closed since another already merged PR #32 does the same thing

**Pull request closed without merging**